extensions/amp-story/1.0/amp-story-desktop-one-panel.css

```diff
@@ -18,4 +18,22 @@
 
 
 .i-amphtml-story-desktop-one-panel {
+  --i-amphtml-story-page-large-desktop-ratio: 69 / 116;
+  --i-amphtml-story-page-small-desktop-ratio: 3 / 4;
+}
+
+.i-amphtml-story-desktop-one-panel amp-story-page {
+  margin: auto !important;
+  aspect-ratio: var(--i-amphtml-story-page-large-desktop-ratio) !important;
```

Comment:
Will this work on iPads? https://caniuse.com/mdn-css_properties_aspect-ratio
should we use the good old padding-top hack to set the aspect ratio instead?

Reply:
Good call. I think we can still use width and height though instead of the padding-top hack.